test/config/test_configurable.rb

```diff
@@ -164,10 +148,115 @@ class Example6
     config_param :obj1, :hash, default: {}
     config_param :obj2, :array, default: []
   end
+
+  module Overwrite
+    class Base
+      include Fluent::Configurable
+
+      config_param :name, :string, alias: :fullname
+      config_param :bool, :bool, alias: :flag
+      config_section :detail, required: false, multi: false, alias: "information" do
+        config_param :address, :string, default: "x"
+      end
+    end
+
+    class Required < Base
+      config_section :detail, required: true do
+        config_param :address, :string, default: "x"
+      end
+    end
+
+    class Multi < Base
+      config_section :detail, required: true do
+        config_param :address, :string, default: "x"
+      end
+    end
+
+    class Alias < Base
+      config_section :detail, alias: "information2" do
+        config_param :address, :string, default: "x"
+      end
+    end
+
+    class DefaultOptions < Base
+      config_section :detail do
+        config_param :address, :string, default: "x"
+      end
+    end
+
+    class DetailAddressDefault < Base
+      config_section :detail do
+        config_param :address, :string, default: "y"
+      end
+    end
+
+    class AddParam < Base
```

Comment:
Oh, I failed to tell what I meant correctly. I didn't mention about any code change at here.
I just want more test cases:
* current `AddParam` changes default value of `address`, and add a parameter `phone_no`
* I want one more test pattern class `AddParam2` to add `phone_no`, without definition about `address`